$OpenBSD$

EXPERIMENTAL: since we don't have RTLD_NOLOAD, replace this with RTLD_LAZY

Index: Utilities/dynamic_library.cpp
--- Utilities/dynamic_library.cpp.orig
+++ Utilities/dynamic_library.cpp
@@ -63,7 +63,7 @@ namespace utils
 #ifdef _WIN32
 		return reinterpret_cast<void*>(GetProcAddress(GetModuleHandleA(lib), name));
 #else
-		return dlsym(dlopen(lib, RTLD_NOLOAD), name);
+		return dlsym(dlopen(lib, RTLD_LAZY), name);
 #endif
 	}
 }
